core/core.gradle.kts

```diff
@@ -9,7 +9,8 @@ checkstyle {
 }
 
 dependencies {
-    api(group = "com.google.guava", name = "guava", version = "27.0.1-jre")
+    implementation(group = "com.google.guava", name = "guava", version = "27.0.1-jre")
+    implementation(group = "com.beust", name = "klaxon", version = "5.0.5")
```

Comment:
I'd split these out into a separate project that is published under a different name.

Alternatively, consider pushing these as a plugin to `klaxon`.

https://github.com/cbeust/klaxon/tree/master/plugins